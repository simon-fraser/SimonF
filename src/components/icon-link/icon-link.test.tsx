import { render } from '@testing-library/react';
import { it, describe, expect } from 'vitest';
import React from 'react';
import IconLink from "./icon-link"

describe('components/icon-link', () => {
  it('will render correctly',() => {
    const {asFragment} = render(<IconLink href="http://example.com/" />)

    expect(asFragment()).toMatchSnapshot();
  })
})