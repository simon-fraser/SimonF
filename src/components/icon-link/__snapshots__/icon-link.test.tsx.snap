// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/icon-link > will render correctly 1`] = `
<DocumentFragment>
  <a
    class="bg-black block duration-200 hover:-translate-y-0.5 hover:bg-gray-800 hover:shadow-xl hover:scale-105 p-1.5 rounded shadow transition"
    href="http://example.com/"
    target="_blank"
  />
</DocumentFragment>
`;
